index.html

```diff
@@ -0,0 +1,63 @@
+<html lang="en">
```

Comment:
- [ ] Kindly add `DOCTYPE` to your HTML document.

Reply:
- [ ] Good job using inline cmments :+1: , However please Remember to explain why you request a change - it might be obvious for you but not for someone else.
for example:=> This is a well-explained request:  **Kindly add DOCTYPE to your HTML document as It is an "information" to the browser about what document type to expect. you can learn more[ about it here ](It is an "information" to the browser about what document type to expect.)**

[OPTIONAL] You can also provide links to the students in some cases, if they need more information :+1: